guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -55,6 +55,23 @@ The Amazon Chime SDK for JavaScript produces several kinds of events on the [Aud
 |------------ | ------------- | ------------- |
 |[metricsDidReceive](https://github.com/aws/amazon-chime-sdk-js/blob/bf6d01e236445684601e24f3e319dede728b5113/src/audiovideoobserver/AudioVideoObserver.ts#L76) | Exposes the WebRTC getStats metrics. There may be differences among browsers as to which metrics are reported. | All |
 
+> Note: We have built a video WebRTC statistics widget in the demo browser meeting app. This widget is shown as an overlay on the video, when the video is enabled.
+>
+> You get the following WebRTC stats in the video stats widget:
+>
+> Upstream video information: Resolution, Bitrate (kbps), Packets Sent, Frame Rate (fps).
+>
+> Downstream video information: Resolution, Bitrate (kbps), Packet Loss (%), Frame Rate (fps).
+>
+> You can check the implementation in the [demo app](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts) to build your own custom widget (look for `getAndShowWebRTCStats` method in the demo). This video stats widget is built using the [getRTCPeerConnectionStats](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideocontroller.html#getrtcpeerconnectionstats) and the [metricsDidReceive](https://github.com/aws/amazon-chime-sdk-js/blob/bf6d01e236445684601e24f3e319dede728b5113/src/audiovideoobserver/AudioVideoObserver.ts#L76) APIs. Currently, there is no direct API in the amazon-chime-sdk-js for a local user to know if the simulcast resolution changes on the remote side. But, based on these stats you are seeing for any particular video tile (remote or local), you can check its resolution, bitrate and then decide if want to put an avatar on that video tile and remove when the resolution gets good or up to your expectation.
+
+
+### Events for monitoring currently active simulast layers
+
+|Event | Notes | Browsers |
+|------------ | ------------- | ------------- |
+|[encodingSimulcastLayersDidChange](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#encodingsimulcastlayersdidchange) | Called when simulcast is enabled and simulcast uplink encoding layers get changed. Check the [simulcast guide](https://aws.github.io/amazon-chime-sdk-js/modules/simulcast.html#receive-upstream-simulcast-layer-change-notification) for more information on simulcast. | Chromium-based |
```

Comment:
Instead of "encoding layers get changed" how about "encoding layers are changed".

Reply:
Yup yup, updated thanks!